packages/payment-processor/src/payment/utils-near.ts

```diff
@@ -0,0 +1,68 @@
+import { BigNumber, BigNumberish, ethers } from 'ethers';
+import { Contract } from 'near-api-js';
+import { Near, WalletConnection } from 'near-api-js';
+import { Near as NearPaymentDetection } from '@requestnetwork/payment-detection';
+
+export async function isValidNearAddress(nearNetwork: Near, address: string) {
+  try {
+    await nearNetwork.connection.provider.query(`account/${address}`, '');
```

Comment:
that seems a very slow way to validate a near address?

Reply:
It's the one Near team pushed so...